### ISSUE TITLE
BatchNormalization issues

### ISSUE THREAD

**Issue by youngwanLEE** [weiliu89/caffe#138]
Hi Wei,

I found a difference about batchNormalization implementation between your prototxt and ResNet-50-deploy.prototxt([link](https://github.com/KaimingHe/deep-residual-networks/blob/master/prototxt/ResNet-50-deploy.prototxt))

In your implementation, batch_norm_param is set to {_ eps : 0.001 } whereas { use_global_stats:true } is set in ResNet-50-deploy.prototxt. 

I wonder which one is correct batchNormalization.

Thank you

**Comment by weiliu89** [comment]
The one that Kaiming He has released is used for testing (deploy.prototxt), so it set use_global_stats to true. During training time, update the mean/variance is slightly better than using the mean/variance stored in the model.

**Comment by youngwanLEE** [comment]
Thank you :)
